mautrix_facebook/user.py

```diff
@@ -986,6 +993,45 @@ async def get_portal_with(self, puppet: pu.Puppet, create: bool = True) -> po.Po
             puppet.fbid, fb_receiver=self.fbid, create=create, fb_type=ThreadType.USER
         )
 
+    async def _create_or_update_space(self):
+        if not self.config["bridge.enable_space_per_user"]:
+            return
+
+        avatar_state_event_content = {"url": self.config["appservice.bot_avatar"]}
+        name_state_event_content = {"name": "Facebook"}  # TODO template
```

Comment:
take a look at https://github.com/beeper/linkedin/blob/master/linkedin_matrix/example-config.yaml#L102-L108 for how I allowed that to be customized in the LinkedIn bridge. I think that doing something similar here would be good

Reply:
This has now been added, as well as changing the config to conform to how the LinkedIn implementation. Hope this is OK :)